cython0.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: cython
```

Comment:
Generally we call these `cython-0` and we can have it `provides: cython=0.x.y`

Reply:
oof, forgot to change `name:` 🤦‍♀️

Reply:
Is this the general format for the provides clause?
```yaml
dependencies:
  provides:
    - cython=0.29.36
```

Reply:
Updated